ipatests/test_integration/test_commands.py

```diff
@@ -1136,3 +1136,33 @@ def test_cacert_manage(self):
             raiseonerr=False
         )
         assert result.returncode == 0
+
+    def test_ipa_adtrust_install_with_locale_issue8066(self):
+        """
+        This test checks that ipa-adtrust-install command runs successfully
+        on a system with locale en_IN.UTF-8 without displaying error below
+        'IndexError: list index out of range'
+        This is a testcase for Pagure issue
+        https://pagure.io/freeipa/issue/8066
+        """
+        # Set locale to en_IN.UTF-8 in .bash_profile to avoid reboot
+        tasks.kinit_admin(self.master)
+        exp_msg = 'en_IN.UTF-8'
+        self.master.run_command(
+            'echo "export LC_TIME=en_IN.UTF-8" >> /root/.bashrc'
+        )
+        self.master.run_command("source /root/.bashrc")
+        result = self.master.run_command('echo "$LC_TIME"')
+        assert result.stdout_text.rstrip() == exp_msg
+        # Install ipa-server-adtrust and check status
+        msg1 = (
+            "Unexpected error - see /var/log/ipaserver-install.log"
+            "for details"
+        )
+        msg2 = "IndexError: list index out of range"
+        tasks.install_packages(self.master, ["*ipa-server-trust-ad"])
+        result = self.master.run_command(
+            ["ipa-adtrust-install", "-U"], raiseonerr=False
+        )
+        assert msg1 not in result.stderr_text
+        assert msg2 not in result.stderr_text
```

Comment:
At the end of the test you will also need to restore the  previous ~/.bashrc file.

Reply:
restored the bashrc file.